### PR TITLE
Release notes updated

### PR DESCRIPTION
**Pull request by cs-yoges-chauriwar**
### Description

Release Notes updated as per `1.0.3` release

### CODE REVIEW

File: widget/release_notes.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 ### What's New
 
-- This widget is now **Certified**
+- Internationalization Support added for Japanese, Korean, and Simplified Chinese languages.
```
Comment (cs-abhishek-shukla) [inline widget/release_notes.md]: Added Internationalization support for Japanese, Korean, and Simplified Chinese languages.

File: widget/release_notes.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
 
-- Following fields now have a tooltip for enhanced understanding and providing input to these fields is now mandatory:
-  - Data Source
-  - Picklist
-  - Picklist Items
+- Widget Filter retained with grid filter on module listing page, when redirected from the Widget through 'View All Records' option.
```
Comment (cs-abhishek-shukla) [inline widget/release_notes.md]: When users return to the records list after viewing a record in detail, the filter conditions they applied are now retained. Previously, returning to the records list cleared the filter conditions applied through the widget's *View All Records* option.

**Review by cs-abhishek-shukla — CHANGES_REQUESTED** [review]

File: widget/release_notes.md (hunk 2 above)
Comment (cs-yoges-chauriwar) [reply widget/release_notes.md]: I think we should keep it like this,

When users return to the records list after viewing a record in detail, the additional grid filter conditions they applied are retained. Previously, returning to the records list widget's _View All Records_ option filter was applied, but the additional grid filter conditions were cleared.

Comment (cs-yoges-chauriwar) [reply widget/release_notes.md]: Done

File: widget/release_notes.md (hunk 1 above)
Comment (cs-yoges-chauriwar) [reply widget/release_notes.md]: Done

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**